packages/store/ts/library/config/storeConfig.ts

```diff
@@ -32,7 +32,7 @@ const zUserEnumName = zObjectName;
 // (user types are refined later, based on the appropriate config options)
 const zFieldData = z.string();
 
-type FieldData<UserTypes extends StringForUnion> = AbiType | StaticArray | UserTypes;
+export type FieldData<UserTypes extends StringForUnion> = AbiType | StaticArray | UserTypes;
```

Comment:
this might be better solved by adding a `FieldType` aggregate to `schema-type`, and having `storeConfig` extend it with `UserTypes`.
You would then want to export/import `schema-types`'s version because I don't think anything outside of config should be aware of UserTypes, or have to deal with an unnecessary generic here

Reply:
tried this but it feels better to import the actual `FieldType` type from store than reconstructing it here (we need to account for enum strings here anyway by mapping them to `uint8`/`number`)